ercc/attestation/.gitignore

```diff
@@ -0,0 +1,3 @@
+ias.go
```

Comment:
this .gitignore seems like a relict to me. With #68 this has been changed and moved to `ercc/`

Reply:
this was an artifact from before your attestation-refactoring-disguised as a test-update overlooked in the merge ;-)  will remove ...